webapp/application.py

```diff
@@ -332,6 +332,16 @@ def _get_gia_feedback(attachments):
     return feedback_attachments
 
 
+def _submitted_email_match(submitted_email, application):
+    candidate_emails_data = application["candidate"]["email_addresses"]
+    candidate_emails_list = [a["value"] for a in candidate_emails_data]
+
+    if submitted_email in candidate_emails_list:
```

Comment:
You can just do `return submitted_email in candidate_emails_list`

Reply:
Good point, done :+1: